src/mumble/AudioInput.cpp

```diff
@@ -213,7 +213,8 @@ bool AudioInputRegistrar::isMicrophoneAccessDeniedByOS() {
 	return false;
 }
 
-AudioInput::AudioInput() : opusBuffer(Global::get().s.iFramesPerPacket * (SAMPLE_RATE / 100)) {
+AudioInput::AudioInput()
+	: m_activeAudioCue(AudioOutputToken::None()), opusBuffer(Global::get().s.iFramesPerPacket * (SAMPLE_RATE / 100)) {
```

Comment:
Shouldn't the none-state essentially be the state in which the token is default-constructed?
In that case we could shorten this a bit to
```suggestion
	: m_activeAudioCue(), opusBuffer(Global::get().s.iFramesPerPacket * (SAMPLE_RATE / 100)) {
```